src/constants.ts

```diff
@@ -14,27 +14,52 @@ export const DTYPE_VALUES = {
     format: GL.R8UI,
     dataFormat: GL.RED_INTEGER,
     type: GL.UNSIGNED_BYTE,
-    max: 2 ** 8 - 1
+    max: 2 ** 8 - 1,
+    sampler: 'usampler2D'
   },
   Uint16: {
     format: GL.R16UI,
     dataFormat: GL.RED_INTEGER,
     type: GL.UNSIGNED_SHORT,
-    max: 2 ** 16 - 1
+    max: 2 ** 16 - 1,
```

Comment:
Let's add a `min` to all of these.

Reply:
Just to push back slightly, why? I don't understand the motivation. A quick `ctl+F` in the codebase shows that we only use `max` once (and I'm not sure if that is necessary).

Reply:
It's helpful for end users in the way we use it in Vitessce.